extraction/extractor.py

```diff
@@ -5,6 +5,7 @@
 
 import requests
 from bs4 import BeautifulSoup, Comment
+from extraction import text_extractor
 from extraction.data_models import contents, text
```

Comment:
Would it be a good idea to group imports (ie. separate different types of imports with a new line)? eg. module imports could be separated from class imports here.

Reply:
Yes sure! I will make the changes.

Reply:
isort does it. Right?

Reply:
Yes! But earlier it was throwing error on adding new line there.

Reply:
Updated accordingly.